README.md

```diff
@@ -166,9 +166,9 @@ var server = require("graphql-server-lambda");
 exports.handler = server.graphqlLambda({ schema: myGraphQLSchema });
 ```
 
-### Zeit Micro
+### ZEIT Micro
 
-Micro function requires the micro module
+Requires the [Micro](github.com/zeit/micro) module
```

Comment:
This is a relative URL; you probably want `[Micro](https://github.com/zeit/micro)`.